docs/source/tutorials/getting-started/intro.rst

```diff
@@ -27,15 +28,35 @@ executing the following command:
    ./get-dependencies.sh
    daml studio
 
-Daml Finance Reference App
-**************************
+Reference App
+*************
 
 In addition to Daml Finance, there also a separate Daml Finance Reference App.
 It showcases several of the Daml Finance capabilites in a web-based graphical user interface.
 
 If you are interested in trying out the Daml Finance Reference App locally, you can clone the corresponding repo
 and follow the installation instructions on the `Daml Finance Reference App GitHub page <https://github.com/digital-asset/daml-finance-app>`_.
 
+
+Create your own App
+*******************
+
+It is possible to generate *JavaScript* code from the Daml Finance packages you need, by running
+daml with the `js` codegen option, for example:
+
+.. code-block:: shell
+
+   daml codegen js -o ./output daml-finance-interface-swap-0.1.7.dar daml-finance-interface-instrument-bond-0.1.7.dar
+
+Alternatively, if your app uses *Java*, you can also run the `java` codegen in a similar way:
+
+.. code-block:: shell
+
+   daml codegen java -o ./output daml-finance-interface-swap-0.1.7.dar daml-finance-interface-instrument-bond-0.1.7.dar
+
+Note, this Daml Finance codegen is only supported on SDK versions 2.5.x and higher.
```

Comment:
For my understanding, what is the fix coming with `2.5.x`?

Reply:
This is a reference to the latest Daml Finance .dars, which contains consistently bumped versions of the packages. It also relates to the feedback on this PR: https://github.com/digital-asset/daml-finance-app/pull/58#discussion_r1018247791

Reply:
Thanks for clarifying, so nothing to do with the SDK really :)